Multisig/Data/Services/Safe Client Gateway Service/SCGModels.swift

```diff
@@ -14,6 +14,68 @@ struct Page<T: Decodable>: Decodable {
     let results: [T]
 }
 
+protocol TransactionSummaryItem: Decodable { }
+
+
+struct TransactionSummaryItemDateLabel: TransactionSummaryItem {
```

Comment:
I wanted to transition from this file's models to the SCG.swift models. Does it make sense to make it in this release or make it as a separate refactoring?